web/src/routes/(app)/account/jobs/+page.server.ts

```diff
@@ -39,45 +34,35 @@ export const load: PageServerLoad<Data> = async ({
 		throw redirect(303, '/login');
 	}
 
-	// get the query parameters from the URL (default to 1)
-	const page = parseInt(url.searchParams.get('page') || '1') || 1;
-	const start = (page - 1) * PAGE_SIZE;
-	const stop = start + PAGE_SIZE;
+	// retreive the total count of the users jobs
```

Comment:
@devstein - this strays a bit from what you described in the initial issue. `getPagePagination` needs the total number of fetched results to compute a number of its properties, so there are two separate calls now: (1) to get the total results before `getPagePagination` and another to get the actual results (only selecting the needed range for the current pagination page) after `getPagePagination`

let me know if this doesn't make sense as is

Reply:
Makes sense. Thanks for calling this out